search/elastic.py

```diff
@@ -17,7 +18,7 @@
 # We _may_ want to use these for their special uses for certain queries,
 # but for analysed fields these kinds of characters are removed anyway, so
 # we can safely remove them from analysed matches
-RESERVED_CHARACTERS = "+-=><!(){}[]^\"~*:\\/&|?"
```

Comment:
have we allowed single quotes `''`?

Reply:
also, @ciuin Where I can observe the behavior of search in prod? 
Is this the endpoint to hit https://www.edx.org/?